projects/portal/src/app/views/dialogs/wallets/ununifi/ununifi-import-wallet-with-mnemonic-form-dialog/ununifi-import-wallet-with-mnemonic-form-dialog.component.ts

```diff
@@ -51,7 +51,7 @@ export class UnunifiImportWalletWithMnemonicFormDialogComponent implements OnIni
             address: '',
           });
         }
-        const mnemonicWithNoWhitespace = mnemonic.trim();
+        const mnemonicWithNoWhitespace = mnemonic.trim().replace(/\s{2,}/g, ' ');
```

Comment:
I noticed that the calculated private key is different if there are two or more spaces in a row.
So, I fix that combined whitespace characters into one.

Reply:
I concern if there are more spaces (ex. 10 spaces), this implementation cannot remove all unnecessary spaces.
My suggestion are as follows.

```suggestion
        const mnemonicWithNoWhitespace = mnemonic.trim().replace(/\s+/g, ' ');
```